zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -102,7 +102,7 @@ pub(super) const TESTNET_ACTIVATION_HEIGHTS: &[(block::Height, NetworkUpgrade)]
     (block::Height(584_000), Blossom),
     (block::Height(903_800), Heartwood),
     (block::Height(1_028_500), Canopy),
-    (block::Height(1_599_200), Nu5),
+    (block::Height(1_821_300), Nu5),
```

Comment:
This is the final height in https://github.com/zcash/zips/commit/9a742b3e6cc160adede4d115a7db87941553957a

```suggestion
    (block::Height(1_842_420), Nu5),
```